scripts/check-merged-worktrees.sh

```
@@ -0,0 +1,55 @@
+
+echo "Checking non-prefixed worktrees against their Shape Up target..."
+echo
+
+git -C "$BARE" worktree list --porcelain | awk '
+  /^worktree / { path=substr($0,10); next }
+  /^branch /  {
+    branch=substr($0,8)
+    print path "\t" branch
+  }
+' | while IFS=$'\t' read -r path branch; do
+  rel="${path#$ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)